data/unique_courses.json

```diff
@@ -1 +1 @@
\ No newline at end of file
+[{"COURSE\r\nCODE":"MGT1022","COURSE NAME":"Lean Start-up Management","COURSE\r\nTYPE":"ETH","CREDITS":"1","VENUE":"SJT823","SLOT":"TE2","FACULTY":"GANGATHARAN C","ALLOTED\r\nSEATS":"70"}]
```

Comment:
Why so less course?
Key names are not as expected. `COURSE CODE` should be just `CODE` and so on for other keys.